src/lib.rs

```diff
@@ -123,34 +123,38 @@
 //! [simd_soundness_bug]: https://github.com/rust-lang/rust/issues/44367
 //! [target_feature_impr]: https://github.com/rust-lang/rust/issues/44839
 
-#![feature(macro_reexport, const_fn, const_atomic_usize_new)]
+#![feature(const_fn, const_size_of, use_extern_macros, cfg_target_feature)]
```

Comment:
I moved the crate from `macro_reexport` to `use_extern_macros` here because `macro_reexport` was behaving weirdly when trying to reuse the macros in `stdsimd`; `use_extern_macros` seems to work way better.